service/geopmdpy/varrun.py

```diff
@@ -190,6 +191,9 @@ def secure_read_file(path):
         if os.path.islink(path):
             sys.stderr.write(f'Warning: <geopm-service> {path} is a symbolic link, it will be renamed to {renamed_path}')
             sys.stderr.write(f'Warning: <geopm-service> the symbolic link points to {os.readlink(path)}')
+        # If it is a fifo
+        elif stat.S_ISFIFO(os.stat(path).st_mode):
```

Comment:
Should check for stat.S_ISREG() before opening it.  This will cover the case of a link or a FIFO.  More verbose error messaging we can check each of these individually first, but I'm also okay with a single check that it is a regular file.

Reply:
Also \n's